Shared/ThresholdKeyView.swift

```diff
@@ -243,11 +244,17 @@ struct ThresholdKeyView: View {
                         Spacer()
                         Button(action: {
                             Task {
-                                let key_details = try! threshold_key.get_key_details()
-                                totalShares = Int(key_details.total_shares)
-                                threshold = Int(key_details.threshold)
-                                alertContent = "There are \(totalShares) available shares. \(key_details.required_shares) are required to reconstruct the private key"
-                                showAlert = true
+                                do {
+                                    let key_details = try threshold_key.get_key_details()
+                                    totalShares = Int(key_details.total_shares)
+                                    threshold = Int(key_details.threshold)
+                                        alertContent = "You have \(totalShares) shares. \(key_details.required_shares) are required to reconstruct the final key\n threshold : \(threshold)\n pub_key: (\(try key_details.pub_key.getX()) , \(try! key_details.pub_key.getY())) \n share_descriptions: \(key_details.share_descriptions)"
```

Comment:
The embedded `try!` can fail here.

Think I said to rather take the previous comment here which does not have this problem:
```suggestion
alertContent = "There are \(totalShares) available shares. \(key_details.required_shares) are required to reconstruct the private key"
                                showAlert = true
```